### ISSUE TITLE
Watch the BOSH Resurrector resurrect Bosh jobs

### ISSUE THREAD

**Issue by chenl23** [pivotal/cf-onboarding#186]
After step "While still SSHed into a VM, locate the agent.json and throw some un-parseable junk in there.", we need to kill the process in order to see resurrection failed due to bad agent.json file.

**Comment by dsabeti** [comment]
Yeah, we should update the story to explain how to restart the BOSH agent.
